subsys/net/ip/rpl.c

```diff
@@ -650,12 +650,18 @@ static void dio_timer(struct k_work *work)
 				net_if_ipv6_get_ll(rpl_default_iface,
 						   NET_ADDR_PREFERRED);
 
-			net_rpl_dio_send(rpl_default_iface, instance, addr,
-					 NULL);
-
+			if (addr) {
+				net_rpl_dio_send(rpl_default_iface, instance,
+						 addr, NULL);
 #if defined(CONFIG_NET_STATISTICS_RPL)
-			instance->dio_send_pkt++;
+	instance->dio_send_pkt++;
```

Comment:
The indentation of this line is wrong, it should be placed in same column as the net_rpl_dio_send() above.

Reply:
Ah! I missed that one. Thanks!